packages/angular/src/schematics/ngrx/schema.json

```diff
@@ -1,59 +1,75 @@
 {
   "$schema": "http://json-schema.org/schema",
   "id": "SchematicsNxNgrx",
-  "title": "Add NgRx support to a module",
+  "title": "Add NgRx support to an application or library",
   "type": "object",
   "properties": {
     "name": {
       "type": "string",
-      "description": "Name of the NgRx feature (e.g., Products, Users, etc.). Recommended to use plural form for name.",
+      "description": "Name of the NgRx feature state, such as \"Products\" or \"Users\"). Recommended to use the plural form of the name.",
```

Comment:
As previously mentioned, should `Products` be title case or all lower case or does it even matter as the schematic will convert to lower?